### PR TITLE
 Slider constraints for super-high res displays, sticky contact & bottom CTA section

### PR DESCRIPTION
**Pull request by rart**

### CODE REVIEW

File: templates/web/product.ftl [hunk 1]
```diff
@@ -164,38 +150,7 @@
       </div>
     </#if>
 
-    <#if contentModel.active?? && contentModel.active == "true">
-      <div class="container clearfix" <@studio.iceAttr iceGroup="final"/>>
-
-      ${contentModel.body_html!""}
```
Comment (sumerjabri) [inline templates/web/product.ftl]: Why are we deleting the body_html element from the footer? Does that mean the product has no HTML section anymore?

File: templates/web/product.ftl [hunk 2]
```diff
@@ -164,38 +150,7 @@
...
-
-        <div class="clear"></div>
-
-        <div class="divider divider-short divider-vshort divider-line divider-center">&nbsp;</div>
-
-          <#if contentModel.showSubscribeWidget?? && contentModel.showSubscribeWidget == "true">
-              <#assign subscribeWidget = siteItemService.getSiteItem(contentModel.subscribeWidget.item.key)/>
-            <div class="heading-block center">
-              <h3>${subscribeWidget.title!""}</h3>
```
Comment (sumerjabri) [inline templates/web/product.ftl]: What about these sections/widgets/etc too?

**Comment by sumerjabri** [comment]
@rart please see my notes and respond within the tool. Thanks.

@russdanner please follow the thread above if you can.

File: templates/web/product.ftl (hunk 1 above)
Comment (rart) [reply templates/web/product.ftl]: This isn't the footer.
Removed per discussion with Russ.

File: templates/web/product.ftl (hunk 2 above)
Comment (rart) [reply templates/web/product.ftl]: Same as above: removed per discussion with Russ.

File: templates/web/product.ftl (hunk 1 above)
Comment (sumerjabri) [reply templates/web/product.ftl]: I meant product. Are you saying that the model is incorrect and this body_html is not needed?

Comment (rart) [reply templates/web/product.ftl]: Yes.

**Comment by sumerjabri** [comment]
@russdanner please update the content model to match what's been changed in the templates per your conversation with Roy

**Pull request merged**